assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/Overview/OptionalCells.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * OptionalCells component for SearchFunnelWidget.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { ProgressBar } from 'googlesitekit-components';
+import { Cell } from '../../../../../../material-components';
+import { CORE_MODULES } from '../../../../../../googlesitekit/modules/datastore/constants';
+import { MODULES_ANALYTICS } from '../../../../../analytics/datastore/constants';
+import { CORE_LOCATION } from '../../../../../../googlesitekit/datastore/location/constants';
+import ActivateAnalyticsCTA from '../ActivateAnalyticsCTA';
+import CreateGoalCTA from '../CreateGoalCTA';
+import RecoverableModules from '../../../../../../components/RecoverableModules';
+import {
+	BREAKPOINT_SMALL,
+	useBreakpoint,
+} from '../../../../../../hooks/useBreakpoint';
+const { useSelect } = Data;
+
+export default function OptionalCells( {
```

Comment:
See the [**Relevant technical choices**](https://github.com/google/site-kit-wp/pull/6436#issue-1549297590) section for a bit of discussion on the decision to extract this component.